mlflow/pyfunc/__init__.py

```diff
@@ -2086,7 +2143,7 @@ def udf_with_default_cols(*args):
 def _validate_function_python_model(python_model):
     if not (isinstance(python_model, PythonModel) or callable(python_model)):
         raise MlflowException(
-            "`python_model` must be a PythonModel instance or a callable object",
+            "`python_model` must be a PythonModel instance, filepath, or a callable object",
```

Comment:
Is `isinstance(python_model, str)` missing?

Reply:
by the time it gets to this statement, the model from the filepath should already be loaded into the var `python_model`

this error message is to not confuse the user (who doesn't have to know about the internal implementation)